examples/computeraytracing/computeraytracing.cpp

```diff
@@ -94,6 +94,9 @@ class VulkanExample : public VulkanExampleBase
 
 	~VulkanExample()
 	{
+		// SRS - Ensure all operations on the device have finished before destroying resources
+		vkDeviceWaitIdle(device);
```

Comment:
This shouldn't be necessary. The base class already does a `vkDeviceWaitIdle` at the end of the render loop.

Reply:
This one is trickier.  On macOS interactive mode exit (not benchmarking exit) does not run through the end of renderLoop().  For the cross-platform `cmake -G Xcode` (VK_EXAMPLE_XCODE_GENERATED) project, the render loop is taken over by `[NSApp run]`.  Interactive termination is handled by the `applicationWillTerminate()` event handler and the exit code of renderLoop() never runs.  The only time it will run is when benchmarking on macOS and `[NSApp run]` is never started.  Your comment points out that I should have placed a `vkDeviceWaitIdle(device)` in the macOS termination event handler instead of in several example cases.  I should be able to fix this quite easily, reverting the example cases.  I can push a small change to the PR but I should test it first.